packages/api-reference/src/components/ApiReferenceBase.vue

```diff
@@ -145,7 +145,21 @@ if (props.configuration?.authentication) {
     {{ currentConfiguration.customCss }}
   </component>
   <ThemeStyles :id="currentConfiguration?.theme" />
-  <FlowToastContainer />
+  <Toaster
+    :toastOptions="{
+      style: {
+        'padding': '18px',
```

Comment:
Can we configure this some other way than a big style block?

Reply:
Yess! Added a class and made a CustomToaster component to hold the styles.